components/Game/index.js

```diff
@@ -44,21 +44,21 @@ const Game = () => {
           </Heading>
         </Row>
 
-        {isReady ? (
-          <PlayerCards
-            cardBeingWatched={!isSelfToPlay ? cardBeingWatched : null}
-            cardPlayerId={player.id}
-            cards={player.cards}
-            onCardHide={handleHideCard}
-            onCardPick={handleCardClick}
-            selectedCards={selectedCards}
-            unfoldedCards={unfoldedCards}
-          />
-        ) : player.isReady ? (
-          <Body>Ready</Body>
-        ) : (
+        <PlayerCards
+          cardBeingWatched={!isSelfToPlay ? cardBeingWatched : null}
+          cardPlayerId={player.id}
+          cards={player.cards}
+          isCardBeingWatched={!isReady}
```

Comment:
This prop doesn't match the one use in `PlayCards` (`isCardBeingWatched` vs `isAllCardsUnfolded`).

We can name this prop `shouldRevealAllCards` to avoid using _is_ with _all_, WDYT?